tests/taskcluster/tasks/raptor.yml

```diff
@@ -34,3 +37,9 @@ task:
         type: directory
         path: '/home/tester/logs'
         expires: '{{nextYear}}'
+
+  extra:
+    treeherder:
+      groupSymbol: Raptor
```

Comment:
Nvm, must have been a treeherder issue, I have since updated the PR and changed the suite name / treeherder item description and it now shows the new group name 'Raptor'.